server/internal/repository/review.go

```diff
@@ -10,3 +15,12 @@ type (
 		Content  string    `db:"content"`
 	}
 )
+
+func (r *Repository) GetEateryEateryIDReviews(ctx context.Context, eateryID uuid.UUID) ([]*Review, error) {
+	reviews := []*Review{}
+	if err := r.db.SelectContext(ctx, reviews, "SELECT * FROM reviews WHERE eatery_id = ?", eateryID); err != nil {
```

Comment:
https://github.com/traP-jp/h25s_04/pull/21#discussion_r2159866738 と同じでここも`&`が必要です！
（`=`を使わずに値を書き換えるにはポインタを使う必要がある、みたいなことを頭の片隅に入れておくと良いです）
```suggestion
	if err := r.db.SelectContext(ctx, &reviews, "SELECT * FROM reviews WHERE eatery_id = ?", eateryID); err != nil {
```